lib/foreman_remote_execution/engine.rb

```diff
@@ -120,7 +120,10 @@ class Engine < ::Rails::Engine
     # content twice.
     assets_to_precompile =
       Dir.chdir(root) do
-        Dir['app/assets/javascripts/**/*', 'app/assets/stylesheets/**/*'].map do |f|
+        Dir[
+          'app/assets/javascripts/foreman_remote_execution/**/*',
+          'app/assets/stylesheets/foreman_remote_execution**/*'
```

Comment:
this was included by accident, will change ince the test run is finished, please don't merge

Reply:
all fine now.